app/src/main/java/kr/co/connect/boostcamp/livewhere/ui/detail/DetailBindingUtil.kt

```diff
@@ -0,0 +1,256 @@
+    }
+}
+
+@BindingAdapter("setBarChart")
+fun setBarChart(barChart: BarChart, list: LiveData<List<HouseAvgPrice>>) {
+    if (!list.value.isNullOrEmpty()) {
+        BarChartUtil.showChart(barChart)
+        BarChartUtil.setChartData(barChart, list.value!!)
+    } else {
+        barChart.setNoDataText(EMPTY_BARCHART_TEXT)
+        barChart.data = null
+    }
+    barChart.notifyDataSetChanged()
+    barChart.invalidate()
+}
+
+@BindingAdapter("setDetailImage") //로드뷰 이미지 세팅
+fun setDetailImage(imageView: AppCompatImageView, location: LiveData<String>?) {
+    if (location != null)
+        Glide.with(imageView.context)
+            .load(
+                imageView.context.getString(
+                    R.string.glide_street_img_url,
+                    location.value,
+                    BuildConfig.GoogleApiKey
+                )
+            )
+            .into(imageView)
+}
+
+@BindingAdapter("setRecentPrice")
+fun setRecentPrice(textView: TextView, recentPrice: LiveData<RecentPrice>) {
+    try {
+        when (textView.id) {
+            R.id.detail_fragment_tv_charter -> {
+                textView.text =
+                    textView.context.getString(R.string.recent_charter_price, recentPrice.value!!.charterPrice)
+            }
+            R.id.detail_fragment_tv_monthly_rent -> {
+                textView.text =
+                    textView.context.getString(R.string.recent_monthly_price, recentPrice.value!!.monthlyPrice)
+            }
+        }
+    } catch (e: KotlinNullPointerException) {
+        when (textView.id) {
+            R.id.detail_fragment_tv_charter -> {
+                textView.text = "전세 정보 없음"
+            }
+            R.id.detail_fragment_tv_monthly_rent -> {
+                textView.text = "월세 정보 없음"
+            }
+        }
+    }
+}
+
+@BindingAdapter("setRvItems") //과거 거래내역 rv
+fun setRvItems(recyclerView: RecyclerView, itemList: List<PastTransaction>?) {
+    if (itemList != null) {
+        (recyclerView.adapter as DetailTransactionRvAdapter).setData(itemList)
+    } else {
+        // TODO 데이터 정보 없음 처리.
+    }
+}
+
+@BindingAdapter("setReviewDelete")
+fun setReviewDelete(imageButton: ImageButton, item: Review) {
+    val pref = SharedPreferenceStorage(imageButton.context)
+    if (item.id == pref.uuid) {
+        imageButton.visibility = View.VISIBLE
+    } else {
+        imageButton.visibility = View.GONE
+    }
+}
+
+@BindingAdapter("setReviews")
+fun setReviews(recyclerView: RecyclerView, reviewList: List<Review>?) {
+    if (!reviewList.isNullOrEmpty()) {
+        (recyclerView.adapter as DetailReviewRvAdapter).setData(reviewList)
+    } else {
+        (recyclerView.adapter as DetailReviewRvAdapter).setData(listOfNotNull())
+    }
+}
+
+@BindingAdapter("setReviewsEmpty")
+fun setReviewsEmpty(textView: TextView, reviewList: List<Review>?) {
+    if (!reviewList.isNullOrEmpty()) {
+        textView.visibility = View.GONE
+    } else {
+        textView.visibility = View.VISIBLE
+    }
```

Comment:
```suggestion
    textView.isVisible = reviewList.isNullOrEmpty()
```
P3. isVisible을 사용하면 한 줄로 표현이 가능합니다.